server/database/queries/selectProjectDetails.js

```diff
@@ -0,0 +1,18 @@
+const connect = require('./../config/connection');
+
+const selectProjectDetails = projectID => connect.query(`SELECT
+                                                          users.user_name, users.full_name, users.role, users.id
+                                                        AS
+                                                          userId, projects.* from users_projects
+                                                        INNER JOIN
```

Comment:
Is this inner join necessary? Why selecting username, fullname and role? Isn't userId enough?
I remember we talked about it yesterday :)